skyportal/plot.py

```diff
@@ -146,38 +146,49 @@ class CheckboxWithLegendGroup(CheckboxGroup):
     "diamond",
     "star",
     "plus",
-    "x",
     "cross",
-    "y",
     "triangle_pin",
     "square_pin",
 ]
 
 
-def get_color(bandpass_name):
-    if bandpass_name.startswith('ztf'):
-        return {'ztfg': 'green', 'ztfi': 'orange', 'ztfr': 'red'}[bandpass_name]
-    else:
-        bandpass = sncosmo.get_bandpass(bandpass_name)
-        wave = bandpass.wave_eff
-
-        if 0 < wave < 3000:
-            cmap = cmap_uv
-            cmap_limits = (0, 3000)
-        elif 3000 <= wave <= 10000:
-            cmap = cmap_opt
-            cmap_limits = (3000, 10000)
-        elif 10000 < wave < 1e5:
-            wave = np.log10(wave)
-            cmap = cmap_ir
-            cmap_limits = (4, 5)
-        else:
-            raise ValueError('wavelength out of range for color maps')
-
+def get_effwave(bandpass_name):
```

Comment:
Does `effwave` mean `effective_wavelength`? If so I'd opt for the latter naming just to be improve readability

Reply:
Good idea.  Recent commit implements this.